parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -189,6 +189,7 @@ apt_get_dist_upgrade() {
     export DEBIAN_FRONTEND=noninteractive
     dpkg --configure -a
     apt-get -f -y install
+    apt-mark showhold
```

Comment:
this doesn't have any functional behavior change but it's nice to have for the logs